src/main/kotlin/Archives.kt

```diff
@@ -0,0 +1,26 @@
+class Archive(titel: String) {
+    val titel = titel
+    var noteList = mutableListOf<Notes>()
```

Comment:
```suggestion
    val noteList = mutableListOf<Notes>()
```
Список и так мьютабельный, делать переменную мьютабельной - излишне